[MTM-50082]: Announcement about a breaking change in Inventory API -- change default behaviour for withChildren parameter (#144)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/announcements-10-17-0.md

* `DELETE /measurements/measurement` requires at least one of the following parameters: `source`, `dateFrom`, `dateTo`

##### Breaking change in the Inventory API -- change of the default value for the `withChildren` parameter

> As of the release 10.20+ a default value of the `withChildren` parameter for `GET /inventory/managedObjects` endpoint will be changed from `true` to `false`.

Reviewer: ```suggestion
As of release 10.20+, the default value for the `withChildren` parameter of the `GET /inventory/managedObjects` endpoint will be changed from `true` to `false`.
```

> It means that when a request should still return child assets, devices and additions, a parameter `?withChildren=true` will have to be provided in the query.

Reviewer: ```suggestion
As a consequence, if a request is supposed to return child assets, devices and additions, a parameter `?withChildren=true` has to be provided in the query.
```

> A change is required to improve the Inventory API performance.

Reviewer: ```suggestion
This change is necessary to improve the Inventory API performance.
```
